src/t8_geometry/t8_geometry_implementations/t8_geometry_linear_axis_aligned.hxx

```diff
@@ -82,6 +82,20 @@ struct t8_geometry_linear_axis_aligned: public t8_geometry_with_vertices
   virtual void
   t8_geom_evaluate_jacobian (t8_cmesh_t cmesh, t8_gloidx_t gtreeid, const double *ref_coords, const size_t num_coords,
                              double *jacobian) const;
+
+  /**
+   * \param[in] forest            The forest of the element.
+   * \param[in] ltreeid           The local tree id of the element's tree
+   * \param[in] element           The element
+   * \param[in] points            points to check
+   * \param[in] num_points        Number of points to check
+   * \param[in, out] is_inside    Array to fill with flags whether the point is inside or not
+   * \param[in] tolerance         Tolerance of the inside-check
+   */
+  virtual void
+  t8_geom_point_batch_inside_element (t8_forest_t forest, t8_locidx_t ltreeid, const t8_element_t *element,
```

Comment:
Why repeat this routine here. It derives from `geometry_with_vertices` where this routine is already implemented as virtual function and you also overwrite it in the associated `cxx` file.